<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingView</class>
 <widget class="QWidget" name="SettingView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>632</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1050</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_13">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QGroupBox" name="gpBSuBarin">
        <property name="minimumSize">
         <size>
          <width>540</width>
          <height>250</height>
         </size>
        </property>
        <property name="title">
         <string>超脑连接配置</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <widget class="QLabel" name="lblSuBrainIP">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>IP</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lblSuBarinPort">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>端口</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lblSuBrainUsername">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>用户名</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lblSuBrainPwd">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>密码</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QLineEdit" name="ledtSuBrainIP">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="ledtSuBrainPort">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="ledtSuBrainUsername">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="ledtSuBrainPwd">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QLabel" name="lblSuBrainChannel1">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>通道1</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lblSuBrainChannel2">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>通道2</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lblSuBrainChannel3">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>通道3</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lblSuBrainChannel4">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>通道4</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QLineEdit" name="ledtSuBrainChannel1">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="ledtSuBrainChannel2">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="ledtSuBrainChannel3">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="ledtSuBrainChannel4">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gpbDB">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>250</height>
         </size>
        </property>
        <property name="title">
         <string>数据库连接配置</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="lblDBIP">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>IP</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lblDBPort">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>端口</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lblDBName">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>数据库名</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lblDBUsername">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>用户名</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lblDBPwd">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>密码</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QLineEdit" name="ledtDBIP">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ledtDBPort">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ledtDBName">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ledtDBUsername">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ledtDBPwd">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="gpbPath">
      <property name="minimumSize">
       <size>
        <width>540</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>路径设置</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>30</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QLabel" name="lblCapturePath">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>抓拍图路径</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lblAvatarPath">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>证件图路径</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lblFacePath">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>人脸子图路径</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QLineEdit" name="ledtCapturePath">
                 <property name="minimumSize">
                  <size>
                   <width>800</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ledtAvatarPath">
                 <property name="minimumSize">
                  <size>
                   <width>800</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ledtFacePath">
                 <property name="minimumSize">
                  <size>
                   <width>800</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <widget class="QPushButton" name="btnCapturePath">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnAvatarPath">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnFacePath">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>60</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnConfirm">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
